website/content/en/preview/AWS/configuration.md

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
I'd love to eventually deprecate this section and maintain an AWS specific docs site and neutral docs site. Can you collapse with tasks/Configuration (and perhaps rename to Operator Configuration)